src/fmv.cpp

```diff
@@ -653,10 +697,7 @@ Fmv::~Fmv()
 void Fmv::Play(const std::string& name)
 {
     auto fmv = mResourceLocator.LocateFmv(mAudioController, name.c_str());
-    if (fmv)
```

Comment:
you deleted a bug fix, probably better to put this back ;) or you need to pull latest code?